packages/wdio-devtools-service/src/auditor.js

```diff
@@ -0,0 +1,131 @@
+import weightedMean from 'weighted-mean'
+
+import Diagnostics from 'lighthouse/lighthouse-core/audits/diagnostics'
+import MainThreadWorkBreakdown from 'lighthouse/lighthouse-core/audits/mainthread-work-breakdown'
+import Metrics from 'lighthouse/lighthouse-core/audits/metrics'
+import TTFBMetric from 'lighthouse/lighthouse-core/audits/time-to-first-byte'
+
+import { quantileAtValue } from './utils'
+import logger from '@wdio/logger'
+
+const log = logger('Auditor')
+
+/**
+ * metric scoring to calculate Lighthouse Performance Score
+ * contains: [media, falloff]
+ * see https://docs.google.com/spreadsheets/d/1_iDW0B870vZF6dAhf1Icdher0gX_KFxCd6Df0_fZ6do/edit#gid=283330180
```

Comment:
Hey,
I don't have access to https://docs.google.com/spreadsheets/d/1_iDW0B870vZF6dAhf1Icdher0gX_KFxCd6Df0_fZ6do/edit#gid=283330180

Can we have some public available url?

Reply:
made if public, it is a document from google anyway